lnrpc/walletrpc/walletkit.proto

```diff
@@ -497,6 +519,42 @@ message ListAddressesResponse {
     repeated AccountWithAddresses account_with_addresses = 1;
 }
 
+message SignMessageWithAddrRequest {
+    // The message to be signed. When using REST, this field must be encoded as
+    // base64.
+    bytes msg = 1;
+
+    // The address which will be used to lookup the private key and sign the
+    // corresponding message
+    string addr = 2;
+}
+
+message SignMessageWithAddrResponse {
+    // The signature for the given message
+    string signature = 1;
```

Comment:
Can add a comment here that the signature is base64 encoded. (same for verifymsg)